bacs/src/main/java/com/adyen/checkout/bacs/internal/ui/DefaultBacsDirectDebitDelegate.kt

```diff
@@ -139,22 +139,24 @@ internal class DefaultBacsDirectDebitDelegate(
         }
     }
 
+    override fun canHandleBackPress() = isModeConfirmation()
+
     override fun handleBackPress(): Boolean {
-        val isConfirmationMode = _componentStateFlow.value.mode == BacsDirectDebitMode.CONFIRMATION
-        return if (isConfirmationMode) {
+        return if (isModeConfirmation()) {
```

Comment:
a small one: i think we should call `canHandleBackPress()` here to check instead of `isModeConfirmation()` directly then we know we will handle back press if we can, then if we want to know how we decide that we can always go see the `canHandleBackPress()` details.